source/osh/ui/view/video/MjpegView.js

```diff
@@ -86,8 +86,12 @@ class MjpegView extends View {
   }
 
   setData(dataSourceId,data) {
+      let imgBlob = new Blob([data.data.frameData]);
+      let url = window.URL.createObjectURL(imgBlob);
+      window.URL.revokeObjectURL(imgBlob);
```

Comment:
I don't think we should revoke the current blob. It could cause issues if the HTML image still makes use of it.
We are correctly revoking the old image blob so it should be enough.

Reply:
Yes you are right, we dont slice anymore so it may be useless

Reply:
exactly

Reply:
done in f9c1cff87cbee9c56ea5b773c6abb9fa407e3519